graphics.c

```diff
@@ -0,0 +1,158 @@
+#include "graphics.h"
+#include "display.h"
+
+static inline void vGraphicsPixelWrite(color pixelColor)
+{
+    vDisplayPixelWrite(pixelColor.colorSeperate.redValue, pixelColor.colorSeperate.greenValue, pixelColor.colorSeperate.blueValue);
+}
+
+
+curserStruct curserA,curserB;
+
+void graphicsInit()
```

Comment:
- Funktionen nach Coding Style benennen: `<Rückgabetyp><Modul><Funktionsname>` - hier also `vGraphicsInit`
- Kommentar hinzufügen
- `vGraphicsInit(void)` - void in Parameterliste explizit angeben